src/test/java/com/google/sps/scheduler/SchedulerTest.java

```diff
@@ -730,7 +730,7 @@ public void outOfTasks() {
     List<ExtendedTask> actualScheduledTasks = scheduler.scheduleInRange(day, farFutureDay);
     List<ExtendedTask> expectedScheduledTasks = Arrays.asList(
         createCustomDurationTaskWithDue(
-            createDateTime(day, Scheduler.START_HOUR, Scheduler.START_MINUTE, ZURICH_TIME_ZONE), TWO_HOURS),
+            createDateTime(day, Scheduler.DEFAULT_START_HOUR, Scheduler.DEFAULT_START_MINUTE, ZURICH_TIME_ZONE), TWO_HOURS),
         createCustomDurationTaskWithDue(
             createDateTime(nextDay, 9, 4, ZURICH_TIME_ZONE), AN_HOUR_AND_A_HALF),
         createCustomDurationTaskWithDue(
```

Comment:
It would probably be helpful to have a simple test with a single even or so and different working hours just to check that it's not scheduled outside of working hours.

Reply:
Agreed.